### DISCUSSION TITLE
Pdf are invalid.

### DISCUSSION THREAD

**Discussion by mvolante21** [community/community#127113] [Repositories]
### Select Topic Area

Product Feedback

### Body

I am trying to upload a PDF, but every time I upload the pdf it keeps giving me an "Invalid" message.  I am currently enrolled in a Coursera course and it requires me to be able to send a successful link for the assignment.  Please, can someone help with why I am unable to properly upload one?

**Comment by HimanthaThathsara** [comment]
Usually PDFs are approved by GitHub but include PDF links that are ignored by GitHub.
That's the last time your file was ignored by GitHub.
